<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="2352" started-at="2019-07-14T03:44:46Z" finished-at="2019-07-14T03:44:49Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="2352" started-at="2019-07-14T03:44:46Z" finished-at="2019-07-14T03:44:49Z">
      <class name="extentReports.LearnExtentReports">
        <test-method status="PASS" signature="startReport()[pri:0, instance:extentReports.LearnExtentReports@1a92922]" name="startReport" is-config="true" duration-ms="1325" started-at="2019-07-14T09:14:46Z" finished-at="2019-07-14T09:14:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startReport -->
        <test-method status="FAIL" signature="demoreportfail()[pri:0, instance:extentReports.LearnExtentReports@1a92922]" name="demoreportfail" duration-ms="66" started-at="2019-07-14T09:14:48Z" finished-at="2019-07-14T09:14:48Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at extentReports.LearnExtentReports.demoreportfail(LearnExtentReports.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- demoreportfail -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:extentReports.LearnExtentReports@1a92922]" name="getResult" is-config="true" duration-ms="838" started-at="2019-07-14T09:14:48Z" finished-at="2019-07-14T09:14:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=demoreportfail status=FAILURE method=LearnExtentReports.demoreportfail()[pri:0, instance:extentReports.LearnExtentReports@1a92922] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="PASS" signature="demoreportpass()[pri:0, instance:extentReports.LearnExtentReports@1a92922]" name="demoreportpass" duration-ms="1" started-at="2019-07-14T09:14:49Z" finished-at="2019-07-14T09:14:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- demoreportpass -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:extentReports.LearnExtentReports@1a92922]" name="getResult" is-config="true" duration-ms="50" started-at="2019-07-14T09:14:49Z" finished-at="2019-07-14T09:14:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=demoreportpass status=SUCCESS method=LearnExtentReports.demoreportpass()[pri:0, instance:extentReports.LearnExtentReports@1a92922] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="PASS" signature="endreport()[pri:0, instance:extentReports.LearnExtentReports@1a92922]" name="endreport" is-config="true" duration-ms="4" started-at="2019-07-14T09:14:49Z" finished-at="2019-07-14T09:14:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endreport -->
      </class> <!-- extentReports.LearnExtentReports -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
